test.py

```diff
@@ -5,33 +5,53 @@
 import ldclient
 from ldclient.config import Config
 
-root = logging.getLogger()
-root.setLevel(logging.INFO)
-ch = logging.StreamHandler(sys.stdout)
-ch.setLevel(logging.INFO)
-formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-ch.setFormatter(formatter)
-root.addHandler(ch)
+# Set sdk_key to your LaunchDarkly SDK key before running
+sdk_key = ""
+
+# Set feature_flag_key to the feature flag key you want to evaluate
+feature_flag_key = "my-boolean-flag"
+
+def show_message(s):
+  print("*** %s" % s)
+  print()
+
+# root = logging.getLogger()
```

Comment:
Why is this commented out?

Reply:
Ah - that should've been deleted. We don't want to override the default log level in these demos.

Reply:
See discussion in the spec page comments.